lib/elixir/lib/integer.ex

```diff
@@ -131,6 +131,9 @@ defmodule Integer do
 
   Raises an error if `base` is less than 2 or more than 36.
 
+  If an integer formatted string wants to be directly converted to a integer,
+  `String.to_integer/1` or `String.to_integer/2` can be used instead.
```

Comment:
Thanks, I totally agree. I found the wording a bit weird too but went for consistency first. But you're right, the boy scout rule should prevail, I'll update both docs.